src/views/ModifyPropertyView.vue

```diff
@@ -0,0 +1,137 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue';
+import { useRouter, useRoute } from 'vue-router';
+import ProjectService from '@/services/ProjectService';
+import InputText from 'primevue/inputtext';
+import Textarea from 'primevue/textarea';
+import Dropdown from 'primevue/dropdown';
+import Button from 'primevue/button';
+
+const props = defineProps<{
+  projectId: string;
+}>();
+
+const router = useRouter();
+const route = useRoute();
+const projectService = new ProjectService();
+
+const id = ref(route.query.propertyId || '');
+const title = ref('');
+const description = ref('');
+const district = ref<string>(''); // Gemarkung
+const corridor = ref<string>(''); // Flur
+const parcel = ref<string>(''); // Flurstück
+const landRegistry = ref<string>(''); // Liegenschaftsbuch
+const usageType = ref<string | null>(null); // Wirtschaftsart
+const usageOptions = [
+  { label: 'GF (Gebäude und Freifläche)', value: 'GF' },
+  { label: 'BF (Betriebsfläche)', value: 'BF' },
+  { label: 'Erholungsfläche', value: 'Erholungsfläche' },
+];
+
+onMounted(() => {
+  if (id.value) {
+    fetchPropertyDetails();
+  }
+});
+
+const fetchPropertyDetails = () => {
+  projectService
+      .getProperty(props.projectId, id.value.toString())
+      .then((property) => {
+        title.value = property.title || '';
+        description.value = property.description || '';
+        district.value = property.district || '';
+        corridor.value = property.corridor || '';
+        parcel.value = property.parcel || '';
+        landRegistry.value = property.landRegistry || '';
+        usageType.value = property.usageType ?? null;
+      })
+      .catch((err) => {
+        console.error('Fehler beim Laden der Objektdetails:', err);
+      });
+};
+
+const updateProperty = () => {
+  if (!id.value) return;
+
+  projectService
```

Comment:
when button 'Speichern' is clicked, the `projectService.updateProperty` (line 58-78) isn't called because the `id` is empty (please refer to [this comment](https://github.com/remsfal/remsfal-frontend/pull/317/files/60bf9f436cae603fd426c35675adff2dad10ed1b#r1894180653))